docker-compose.yml

```diff
@@ -58,12 +57,13 @@ services:
       - UNLEASH_PORT=${UNLEASH_PORT-4242}
       - UNLEASH_LOG_LEVEL=${UNLEASH_LOG_LEVEL-WARNING}
       - CURRENCY_URL=${CURRENCY_URL}
-    privileged: true
     ports:
       - 8000:8000
       - 8001:9000
     volumes:
-      - '.:/koku/'
+      - './koku:/koku/koku'
```

Comment:
Wouldn't this make `${KOKU_PATH}/scripts` unavailable? I feel like I'm missing something here.

Reply:
Do the docker-compose services need access to the scripts dir?

Reply:
If the scripts dir is unavailable, how is it executing the `run_migrations.sh` script?

Reply:
Well, it would be an issue if you were making changes to the run_migrations.sh script, that is. You'd have to rebuild the base image every time.

Reply:
I can add it back, but locally we don't rely on the `run_migrations.sh`. The koku-server is started via `run_server.sh` which runs the migrations for us.